<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>541</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>78</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="3">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="b_label">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>B:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="L_label">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>L:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="H_label">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>H:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" rowspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="3">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="RMS_label">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>rms2D:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="hdop_label">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>HDOP:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="vdop_label">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>VDOP:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="3" rowspan="3">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="2" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="speed_label">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>Speed:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="course_label">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>Course:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="6" rowspan="3">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QLabel" name="GPS_label">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string>GPS:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8" rowspan="3">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QLabel" name="time_label">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string>Time:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QLabel" name="GLN_label">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string>GLN:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="9">
        <widget class="QLabel" name="date_label">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string>Date:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="7">
        <widget class="QLabel" name="GLL_label">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string>GLL:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="9">
        <widget class="QLabel" name="status_label">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuInformation">
    <property name="title">
     <string>Information</string>
    </property>
    <addaction name="actionShow_logs"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Plugins</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuInformation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>QA Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSearch"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionShow_logs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="serialBar">
   <property name="windowTitle">
    <string>Serial port</string>
   </property>
   <property name="toolTip">
    <string>Serial port configuration</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionShow_logs">
   <property name="text">
    <string>Show logs</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
   <property name="toolTip">
    <string>Sync with navigation receiver</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
